content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -538,6 +539,163 @@ Unlike the first remote cluster, the second and subsequent clusters added to the
 provide mesh config, but instead are only sources of endpoint configuration, just like remote clusters in a
 [primary-remote](/docs/setup/install/multicluster/primary-remote_multi-network/) Istio multicluster configuration.
 
-### Mesh operator instructions
+To proceed, you'll need another Kubernetes cluster for the second remote cluster of the mesh. Set the following
+environment variables to the context name and cluster name of the cluster:
+
+{{< text syntax=bash snip_id=none >}}
+$ export CTX_SECOND_CLUSTER=<your second remote cluster context>
+$ export SECOND_CLUSTER_NAME=<your second remote cluster name>
+{{< /text >}}
+
+### Register the new cluster
+
+1. Create a secret with credentials to allow the control plane to access the endpoints on the second remote cluster
+    and install it:
+
+    {{< text bash >}}
+    $ istioctl x create-remote-secret \
+      --context="${CTX_SECOND_CLUSTER}" \
+      --type=remote \
+      --namespace=external-istiod | \
+      kubectl apply -f - --context="${CTX_REMOTE_CLUSTER}"
+    {{< /text >}}
+
+    Note that unlike the first remote cluster of the mesh, which also serves as the config cluster, the `--type` argument
+    is set to `remote` this time, instead of `config`. Also note that the secret is applied in the remote (config)
+    cluster, instead of the external cluster, because that is where the external istiod is watching for additions.
+
+1. Create the remote Istio install configuration, which installs the injection webhook that uses the
+    external control plane's injector, instead of a locally deployed one:
+
+    {{< text syntax=bash snip_id=get_second_config_cluster_iop >}}
+    $ cat <<EOF > second-config-cluster.yaml
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    metadata:
+      namespace: external-istiod
+    spec:
+      profile: empty
+      components:
+        istiodRemote:
+          enabled: true
+      values:
+        pilot:
+          configMap: false
+        telemetry:
+          enabled: false
+        global:
+          omitSidecarInjectorConfigMap: true
+          istioNamespace: external-istiod
+        istiodRemote:
+          injectionURL: https://${EXTERNAL_ISTIOD_ADDR}:15017/inject/cluster/${SECOND_CLUSTER_NAME}/net/network2
```

Comment:
Note that passing env variables through the URL path this way doesn't work for values containing / chars.